app/controllers/application_controller.rb

```diff
@@ -1,2 +1,15 @@
+# frozen_string_literal: true
+
 class ApplicationController < ActionController::API
+  private
+
+  def authenticate_request
+    @decoded = Auth::JwtWebToken.decode(request.headers['Auth'])
+  rescue JWT::DecodeError, JWT::VerificationError
+    nil
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: authenticate_request&.fetch(:user_id, nil))
+  end
 end
```

Comment:
Perhaps it would be better to force the service to perform these actions or simple_command gem in order not to clutter the controller from the beginning?